### DISCUSSION TITLE
Noob getting 404 error

### DISCUSSION THREAD

**Discussion by BugOps23** [community/community#22084] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve gone through countless post and guides redoing it and trying to figure it out on my own but I’m at a loss. Can I be pointed in the right direction please :frowning: . <a href="https://github.com/BugOps23/BugOps23.github.io" rel="nofollow noopener">https://github.com/BugOps23/BugOps23.github.io</a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>This can happen for a few reasons, but the most likely one is that this repository was created as a private repo originally and then changed to public afterwards. If you’re on the GitHub free plan you can’t create Pages sites from private repositories, so Pages isn’t built automatically when you push a commit to a private repo.</p>
<p>The quickest way to fix this in future is to push a fresh commit to your repository after changing your visibility settings to public.</p>
<p>I’ve gone ahead and triggered a build for your site and can see it online now!</p>

**Comment by BugOps23** [comment]
<p>Thank you, Thomas. You’re the real mvp.</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/bugops23">@BugOps23</a> , we’re glad you’re here :grinning:</p>
<p>However, we don’t want to have job listings on the Community Forum right now.<br>
Thanks for your interest :+1:</p>
